app/src/main/java/org/simple/clinic/storage/monitoring/DbMethodLookup.kt

```diff
@@ -0,0 +1,46 @@
+package org.simple.clinic.storage.monitoring
+
+import androidx.annotation.VisibleForTesting
+import androidx.annotation.VisibleForTesting.PRIVATE
+
+class DbMethodLookup @VisibleForTesting(otherwise = PRIVATE) constructor(
+    metadataCsv: String
+) {
+
+  private val allDaoMethods: List<DaoMetadata> = DaoMetadata.parse(metadataCsv)
```

Comment:
Just a personal preference, we can remove the explicit type?

Reply:
Yeah, I will make that change in the final PR that integrates the build tooling.